doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -332,6 +332,9 @@ Modem libraries
   * Added:
 
     * Support for the application to send the Wi-Fi access point list to the cloud.
+    * Introduced :kconfig:option:`CONFIG_LOCATION_SERVICE_EXTERNAL`, which replaces the following configurations that are removed:
+      ``CONFIG_LOCATION_METHOD_GNSS_AGPS_EXTERNAL``, ``CONFIG_LOCATION_METHOD_GNSS_PGPS_EXTERNAL`` and ``CONFIG_LOCATION_METHOD_CELLULAR_EXTERNAL``.
```

Comment:
```suggestion
      * ``CONFIG_LOCATION_METHOD_GNSS_AGPS_EXTERNAL``
      * ``CONFIG_LOCATION_METHOD_GNSS_PGPS_EXTERNAL``
      * ``CONFIG_LOCATION_METHOD_CELLULAR_EXTERNAL``
```

Reply:
I guess we need an empty line before the list or otherwise it's just normal text, not bullet list. I'll add that manually.

Reply:
You are so right.